TombEngine/Objects/Effects/flame_emitters.cpp

```diff
@@ -143,23 +170,17 @@ namespace TEN::Entities::Effects
 				if (item->TriggerFlags < 8)
 					FlameEmitterFlags[item->TriggerFlags] = true;
 
-				AddFire(item->Pose.Position.x, item->Pose.Position.y, item->Pose.Position.z, item->RoomNumber, 1.0f, 0);
+				AddFire(item->Pose.Position.x, item->Pose.Position.y, item->Pose.Position.z, item->RoomNumber, 2.0f, 0);
 
 				TriggerDynamicLight(item->Pose.Position.x, item->Pose.Position.y, item->Pose.Position.z,
-					16 - (GetRandomControl() & 1),
-					(GetRandomControl() & 0x3F) + 192,
-					(GetRandomControl() & 0x1F) + 96, 0);
+									16 - (GetRandomControl() & 1),
```

Comment:
why this big line spacing?

Reply:
It looks like GH bug, in both Notepad++ and VS, line spacing is okay.